Source/SwiftyDropboxObjC/Shared/Handwritten/DBXDropboxAppClient.swift

```diff
@@ -0,0 +1,29 @@
+///
+/// Copyright (c) 2022 Dropbox, Inc. All rights reserved.
+///
+
+import Foundation
+import SwiftyDropbox
+
+/// The client for the App API. Call routes using the namespaces inside this object (inherited from parent).
+
```

Comment:
nit- is this extra line preventing the comment from being associated with the class? (If you option-click DBXDropboxAppClient, do you see the comment?)

Reply:
It seems to still work but might as well make this all standard anyway